oaknational/Oak-Web-Application
# LESQ-1106: fix for broken canonical share pages

## Description

Music year: 1963

- passes down lesson pathways to share button, in the even there's no programme slug, finds the first available slug to navigate to share page

#### N.B. Fixes to the canonical pages are probably required in a few areas

New ticket created to make a canonical share page - https://www.notion.so/oaknationalacademy/I-want-a-canonical-Share-page-11a26cc4e1b1801999f8e891ce8000ff

## Issue(s)

Fixes #LESQ-1106

## How to test

1. Go to https://deploy-preview-2859--oak-web-application.netlify.thenational.academy
2. Navigate to any canonical pages (including KS4 Maths and Science), share button should link to a share page

## Screenshots

https://github.com/user-attachments/assets/b73643ef-196b-4440-b6bf-ad18ac25b6a5




## Checklist
